catalog/ftp/src/main/java/ddf/catalog/ftp/FtpServerStarter.java

```diff
@@ -115,23 +124,37 @@ public void destroy() {
      * @param properties map of configurable properties
      */
     public void updateConfiguration(Map<String, Object> properties) {
+        if (MapUtils.isEmpty(properties)) {
+            LOGGER.error("Received null or empty configuration during refresh.");
```

Comment:
warn, and can you change the message so that it is something like "Received null or empty FTP configuration. Check the 'FTP Endpoint Configuration' in the Admin Console" or something along those lines.